docs/src/architectures/pet.rst

```diff
@@ -25,108 +25,81 @@ This will install the PET model along with the ``metatrain`` package.
 Default Hyperparameters
 -----------------------
 
-The default hyperparameters for the PET model are:
+The description of all the hyperparameters used in PET is provided further
+down this page. However, here we provide you with a yaml file containing all 
+the default hyperparameters, which might be convenient as a starting point to
+create your own hyperparameter files:
 
 .. literalinclude:: ../../../src/metatrain/pet/default-hypers.yaml
    :language: yaml
+   :lines: 2-
```

Comment:
Shouldn't this be removed. As default values should be shown by the new class?

Reply:
Aaaah you mean the whole file, I thought you meant the `lines` thing :sweat_smile: I think it is nice to have the yaml file that users can just copy, no?